src/mpi.c

```diff
@@ -2650,7 +2650,7 @@ int mp_exptmod_fast (mp_int * G, mp_int * X, mp_int * P, mp_int * Y, int redmode
 {
   mp_int  M[TAB_SIZE], res;
   mp_digit buf, mp;
-  int     err, bitbuf, bitcpy, bitcnt, mode, digidx, x, y, winsize;
+  int     err, bitbuf, bitcpy, bitcnt, mode, digidx, x, y, winsize, winsize_shift, winsize_minus_shift;
```

Comment:
this file comes from libtommath. I would be more comfortable disabling warnings for this file rather than mutating it. in fact, it may even be due for an update.